controllers/movieController.js

```diff
@@ -0,0 +1,22 @@
+const movieService = require('../services/movieService');
+
+const getMovies = async (req, res) => {
```

Comment:
```suggestion
const getMovies = catchAsync((req, res) => {
```
utils에 작성되어 있는 error.js 내부의 모듈들을 사용해주세요.